eth/db/chain.py

```diff
@@ -186,7 +188,9 @@ def get_block_uncles(self, uncles_hash: Hash32) -> List[BlockHeader]:
 
     # TODO: This method should take a chain of headers as that's the most common use case
     # and it'd be much faster than inserting each header individually.
```

Comment:
cc @gsalgado just remembering we have this pending optimization that we can still do.

Reply:
Jumped on it https://github.com/ethereum/py-evm/pull/1192